### DISCUSSION TITLE
Swarm swarm (June 16, 2024) meta-discussion

### DISCUSSION THREAD

**Discussion by kostmo** [swarm-game/swarm#1931] [General]
## Suggestions for discussion
* Procedural generation: beyond Perlin noise ([article](https://heredragonsabound.blogspot.com/2019/02/perlin-noise-procedural-content.html))
* Developing level content (scenarios)
    * [techniques](https://www.reddit.com/r/gamedev/comments/181ivzq/comment/kadnv9y/?share_id=toJXWBimy2o4UFJqRqHEq&utm_content=2&utm_medium=android_app&utm_name=androidcss&utm_source=share&utm_term=2)
    * Design principles?
    * Registry of game mechanics
* Utilizing tournament server
* Generalized callback mechanism (extending the "watch" command)
* Spinoff opportunities
    * [Gloss demo](https://discourse.haskell.org/t/grid-cycle-gloss-minimal-scaffold-for-a-grid-based-game-in-gloss/9541)
* Issue resolution wishlist top 3
    * Mine: #681
        * You're getting your wish: #1928 !

**Comment by byorgey** [comment]
We should also discuss putting out an 0.6 release.
